docs/Hermes-Relayer-Setup.md

```diff
@@ -66,21 +66,28 @@ Switch to the `hermes` user.
 su hermes
 ```
 
-Restore the account key for the chains you want to relay to. Each key name must match the chain ID, in this example they are `chain-1` and `chain-2`.
+Restore the account key for the chains you want to relay to. Each key name must match the chain ID, in this example they are `my-chain-1` and `my-chain-2`.
 ```
-~/bin/hermes -c ~/.hermes/config.toml keys restore chain-1 -m "abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon art"
+~/bin/hermes -c ~/.hermes/config.toml keys restore my-chain-1 -m "abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon art"
 
-~/bin/hermes -c ~/.hermes/config.toml keys restore chain-2 -m "abandon cabbage abandon cabbage abandon cabbage abandon cabbage abandon cabbage abandon cabbage abandon cabbage abandon cabbage abandon cabbage abandon cabbage abandon cabbage abandon garage"
+~/bin/hermes -c ~/.hermes/config.toml keys restore my-chain-2 -m "abandon cabbage abandon cabbage abandon cabbage abandon cabbage abandon cabbage abandon cabbage abandon cabbage abandon cabbage abandon cabbage abandon cabbage abandon cabbage abandon garage"
 ```
 
 Create a client between the chains.
 ```
-~/bin/hermes -c ~/.hermes/config.toml create client chain-1 chain-2
+~/bin/hermes -c ~/.hermes/config.toml create client my-chain-1 my-chain-2
 ```
 
-Create a channel between the chains.
+Create a connection between the chains.
 ```
-~/bin/hermes -c ~/.hermes/config.toml create channel --port-a transfer --port-b transfer chain-1 chain-2
+~/bin/hermes -c ~/.hermes/config.toml create connection my-chain-1 my-chain-2
+```
+
+Note down the `ConnectionId` returned.
+
+Create a channel between the chains using the chain name and `ConnectionID` above for this example we use `connection-0`
+```
+~/bin/hermes -c ~/.hermes/config.toml create channel --port-a transfer --port-b transfer my-chain-1 connection-0
```

Comment:
Can you add this note after the code block?
Note down save the `ChannelId` for both chains. You will need it whenever you want to make IBC transfers, as shown below.